crates/core/tedge/src/system_services/managers/general_manager.rs

```diff
@@ -213,7 +213,7 @@ impl ServiceCommand<'_> {
             ),
             Self::Start(service) => ExecCommand::try_new_with_placeholder(
                 service_manager.init_config.start.clone(),
-                ServiceCommand::Enable(service),
+                ServiceCommand::Start(service),
```

Comment:
I believe the following should work since `ServiceCommand: Copy`. Assuming it does, that will do a better job of preventing these errors.

```suggestion
                self,
```

Reply:
It works. Indeed less error prone.